packages/palette/src/elements/Typography/Typography.tsx

```diff
@@ -186,6 +198,7 @@ function createStyledText<P extends StyledTextProps>(
             fontFamilyType && themeProps.fontFamily[fontType][fontFamilyType]
           }
           {...determineFontSizes(fontType, size)}
+          {...(unstable_trackIn ? { letterSpacing: "-0.03em" } : {})}
```

Comment:
instead of setting the prop via the mixin up above, we can lock it down a bit:

Up above where we pluck out the props, can do:
```
...
unstable_trackIn,
element,
style,
...textProps
...
```
(note how we're now pulling out `style`)

And then here
```
style={{
  ...(unstable_trackin ? { letterSpacing: "-0.03em" } : {}) 
  ...style
}}
```
Hope this makes sense...